egui_glium/src/painter.rs

```diff
@@ -241,24 +241,6 @@ impl Painter {
         id
     }
 
-    /// register glium texture as egui texture
-    /// Usable for render to image rectangle
-    pub fn register_glium_texture(
-        &mut self,
-        texture: glium::texture::SrgbTexture2d,
-    ) -> egui::TextureId {
-        let id = self.alloc_user_texture();
-        if let egui::TextureId::User(id) = id {
-            if let Some(Some(user_texture)) = self.user_textures.get_mut(id as usize) {
-                *user_texture = UserTexture {
-                    pixels: vec![],
-                    gl_texture: Some(texture),
-                }
-            }
-        }
-        id
-    }
-
```

Comment:
Please add back the old method with a deprecation notice

``` rust
    #[deprecated = 'Use NativeTexture::register_native_texture` instead]
    pub fn register_glium_texture(
        &mut self,
        texture: glium::texture::SrgbTexture2d,
    ) -> egui::TextureId {
        use NativeTexture as _;
        self.register_native_texture(texture)
    }
```

Reply:
ok i will add depecated notice